introduction.md

```diff
@@ -7,4 +7,5 @@ My favourite Code School path is Javascript.
 * Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?